docs/topics/api/reviewers.rst

```diff
@@ -165,7 +165,7 @@ This endpoint allows you to browse through the contents of an Add-on version.
     :>json string validation_url: The absolute url to the addons-linter validation report, rendered as HTML.
     :>json boolean has_been_validated: ``True`` if the version has been validated through addons-linter.
     :>json object addon: A simplified :ref:`add-on <addon-detail-object>` object that contains only a few properties: ``id``, ``name``, ``icon_url`` and ``slug``.
-    :>json object file: The file attached to this version. See :ref:`version detail -> files[] <version-detail-object>` for more details.
+    :>json object file: The file attached to this version. Only includes ``id`` from :ref:`version detail -> files[] <version-detail-object>`.
```

Comment:
I don't understand this - it doesn't only include `id` - it includes a bunch of other fields too.

Reply:
Are you sure about that? Which fields? The only one I can see from https://addons-server.readthedocs.io/en/latest/topics/api/addons.html#version-detail-object other than `id` is `size`, but that's not the same field. It's not the size of the xpi file, but the size of the file requested. Maybe I'm misunderstanding what you're saying, but my intention was to not return any of the `file` fields from `version.file` other than `id`.

Reply:
Oh, you mean _of the fields from `version-details-object`_ it only includes `id`.  That's accurate, but, imo, confusing - better to just include the description for `id` in the block below.